HW03/tim_bulat/spec/tim_bulat_hw_03_t_03_rspec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require_relative '../tim_bulat_hw_03_t_03'
+
+RSpec.describe Homework3 do
```

Comment:
1. the same thing with the subject
2. let's try to come up with more specific names for the variables